Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.7.1
+FROM golang:1.7.3
```

Comment:
Looks like it's https://github.com/golang/go/issues/14481 - so for our CI tests, which run with `-race` enabled, I don't think we can use alpine :|